### PR TITLE
Reimplement weather

### PR DESCRIPTION
**Pull request by MisaOgura**

### CODE REVIEW

File: app/controllers/weathers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  before_action :set_weather, only: [:show, :update, :destroy]
+
+  def index
+    @weather = get_weather
+    render json: @weather
+  end
+
+  private
+
+  def get_weather
```
Comment (shaneoston72) [inline app/controllers/weathers_controller.rb:13]: three methods here....  perhaps private?

File: app/controllers/weathers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    weather = WeatherApi.new.grab_json
+    desc = weather['weather'][0]['description']
```
Comment (shaneoston72) [inline app/controllers/weathers_controller.rb]: desc?  what is this? better name?

**Comment by rhiannonruth** [comment]
🚢

**Comment by shaneoston72** [comment]
🚢

**Comment by Managram** [comment]
:shipit:

**Pull request merged**